src/u2f_device.c

```diff
@@ -240,6 +243,14 @@ static void u2f_device_authenticate(const USB_APDU *a)
         return;
     }
 
+#ifndef BOOTLOADER
```

Comment:
Good catch. It actually was getting compiled when building the bootloader but is not needed. Will fix.